fv3core/.jenkins/actions/run_regression_tests.sh

```diff
@@ -2,8 +2,17 @@
 set -e -x
```

Comment:
Can you base this PR off of the feature/dace-orchestrate-delnflux branch, since we only plan to run it in branches based off that branch (or main after it's been merged into main)?